### DISCUSSION TITLE
UPGRADE from 4.0.1 to 4.1.11

### DISCUSSION THREAD

**Discussion by prwebandprint** [openstore-ecommerce/OpenStore#259] [Q&A]
Just a quick question is there any issues upgrading from 4.0.1 to 4.1.11 on DNN v. 09.04.04 (5)

Thanks,
In Advance

**Comment by DNNMonster** [comment]
I have not performed that leap but I am not aware of an issue.  You will need to back up your file system and db before you do it but I say go for it.  If you're able to do a test upgrade on a non production environment you could also do that to be super cautious.
